src/wp-includes/cache-compat.php

```diff
@@ -141,3 +141,37 @@ function wp_cache_flush_runtime() {
 		return wp_using_ext_object_cache() ? false : wp_cache_flush();
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_group' ) ) :
+	/**
+	 * Removes all cache items in a group.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @see WP_Object_Cache::flush_group()
+	 * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+	 *
+	 * @param string|array $group name(s) of group to remove from cache.
+	 *
+	 * @return bool|Array True or array of bool in array passed on success, false on failure group not found.
+	 */
+	function wp_cache_flush_group( $group ) {
+
+		global $wp_object_cache;
+
+		// if group is an array loop and call each key in the array
+
+		if ( is_array( $group ) ) {
+			$result   = false;
+			$result[] = array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+			return $result;
+		}
+
+		if ( method_exists( $wp_object_cache, 'flush_group' ) ) {
+			return $wp_object_cache->flush_group( $group );
+		}
+
+		return false;
```

Comment:
The compat method should probably just flush the entire cache. That's the safest failover.

```suggestion
		return wp_cache_flush();
```